noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -334,8 +334,7 @@ pub global PROPOSED_BLOCK_HEADER_LENGTH_BYTES: u32 = (PROPOSED_BLOCK_HEADER_LENG
     + 8 /* timestamp */
     + 20 /* coinbase */
     + 16 /* gas_fees.fee_per_da_gas */
-    + 16 /* gas_fees.fee_per_l2_gas */
-;
+    + 16 /* gas_fees.fee_per_l2_gas */;
```

Comment:
Note: This is an unrelated change - `contants.in.ts` was unable to read this constant because the `;` was on a new line